pom.xml

```diff
@@ -29,7 +29,7 @@
     <changelist>-SNAPSHOT</changelist>
     <gitHubRepo>jenkinsci/blueocean-plugin</gitHubRepo>
     <java.level>8</java.level>
-    <jenkins.version>2.176.4</jenkins.version> <!-- Should be kept in sync with the version resulting from Dockerfile FROM statement-->
+    <jenkins.version>2.222.4</jenkins.version>
```

Comment:
just figured out this upgrade is not so trivial... :( because the Jenkinsfile need to be upgraded as well to have BO ATH running with this version of Jenkins. And this not working due to some issue with https://github.com/jenkinsci/java-client-api. I will try to figure out this next week.